universal-application-tool-0.0.1/app/controllers/applicant/ApplicantProgramBlocksController.java

```diff
@@ -208,6 +208,60 @@ public ApplicantProgramBlocksController(
         .exceptionally(ex -> handleUpdateExceptions(ex));
   }
 
+  /**
+   * This method is used by the file upload question. If an user chooses to skip uploading a file,
+   * they click the skip button on the page which takes them to this endpoint. This method puts an
+   * empty file key which marks the file quesiton as seen but not answered and redirects the user to
+   * the next incomplete block.
+   */
+  @Secure
+  public CompletionStage<Result> skipFile(
+      Request request, long applicantId, long programId, String blockId, boolean inReview) {
```

Comment:
suggestion (blocking): please test this method in `ApplicantProgramBlocksControllerTest`

Reply:
Done.